src/pretix/static/pretixcontrol/scss/main.scss

```diff
@@ -36,10 +36,20 @@ nav.navbar {
     margin: 0;
     border: 0;
 }
+.navbar-brand {
+  position: relative;
+  padding-bottom: 10px;
+  padding-top: 10px;
```

Comment:
I think the logo sits to far to the right:

<img width="250" height="107" alt="Bildschirmfoto 2025-11-27 um 10 07 31" src="https://github.com/user-attachments/assets/8b00e26c-0fe6-445c-a76d-5608c1a29df2" />

IMHO, it should sit like this:

<img width="255" height="112" alt="Bildschirmfoto 2025-11-27 um 10 07 23" src="https://github.com/user-attachments/assets/3286f776-970b-4cdc-a41b-f8cfd0f35685" />

```suggestion
  padding-top: 10px;
  padding-left: 10px;
```